app/views/schools/cohort_setup/appropriate_body_type.html.erb

```diff
@@ -15,7 +15,7 @@
                 legend: { text: title, tag: :h1, size: 'xl' },
                 caption: { text: @school.name, size: 'xl' }) do %>
 
-                <p class="govuk-body">If your school works with more than one, first tell us about the appropriate body appointed for the majority of your ECTs. You can add the others later.</p>
+                <p class="govuk-body">If any of your ECTs will work with a different appropriate body, you can tell us that when you register them later.</p>
```

Comment:
Is "ECTs will work with" the right language? I thought the role of appropriate bodies was more to check the programme and formally sign off the final induction? I might well be wrong though! Do they have much/any direct contact with the early career teachers?

Reply:
Hi Frankie, there's some information here: https://www.gov.uk/guidance/guidance-for-early-career-teachers-ects-ecf-based-training#if-you-have-difficulties-or-concerns. Thanks